packages/trader/src/AppV2/Utils/contract-details-config.ts

```diff
@@ -0,0 +1,45 @@
+import { CONTRACT_TYPES } from '@deriv/shared';
+
+const DEFAULT_CONFIG = {
+    is_take_profit_visible: false,
+    is_deal_cancellation_visible: false,
+    is_stop_loss_visible: false,
+    is_tp_history_visible: false,
+};
+
+export const getContractDetailsConfig = (contract_type: string) => {
+    const config = {
+        [CONTRACT_TYPES.ACCUMULATOR]: {
+            is_take_profit_visible: true,
+            is_deal_cancellation_visible: false,
+            is_stop_loss_visible: false,
+            is_tp_history_visible: true,
+        },
+        [CONTRACT_TYPES.MULTIPLIER.UP]: {
+            is_take_profit_visible: true,
+            is_deal_cancellation_visible: true,
+            is_stop_loss_visible: true,
+            is_tp_history_visible: true,
+        },
+        [CONTRACT_TYPES.MULTIPLIER.DOWN]: {
+            is_take_profit_visible: true,
+            is_deal_cancellation_visible: true,
+            is_stop_loss_visible: true,
+            is_tp_history_visible: true,
+        },
+        [CONTRACT_TYPES.TURBOS.LONG]: {
+            is_take_profit_visible: true,
+            is_deal_cancellation_visible: false,
+            is_stop_loss_visible: false,
+            is_tp_history_visible: true,
+        },
+        [CONTRACT_TYPES.TURBOS.SHORT]: {
+            is_take_profit_visible: true,
+            is_deal_cancellation_visible: false,
+            is_stop_loss_visible: false,
+            is_tp_history_visible: true,
```

Comment:
What do you think about renaming all properties using `camelCase`? )

Reply:
Well, since you're asking) It offers absolutely zero benefits over any other case)

Reply:
It's about style agreement that we have)

Reply:
Well, I haven't seen or signed any agreement haha, and to be honest, I'd never agreed to anything meaningless like that) We'll make the changes though, as there's no other way to get it approved haha.

Reply:
done thanks

Reply:
@akmal-deriv it's in the README unfortunately